### DISCUSSION TITLE
Fracture jewel's

### DISCUSSION THREAD

**Discussion by kattpess** [xanthics/PoE_Weighted_Search#36] [Ideas]
i noticed when im searching for "Abyss jewels". he dont search for fracture stats. 

we tried putting in same filter stats only with Fractured infront. with same  weight. and we found alot more jewels. 
i

**Comment by xanthics** [comment]
Explicit mods find fractured mods too.  This was changed ~3-4 leagues ago.  Clickable link for searches.

[fracture life](https://www.pathofexile.com/trade/search/Expedition/JLWwpoRcl)
![image](https://user-images.githubusercontent.com/3137623/131978158-8fe1a5b5-a99e-4c0b-a838-64eaa5c164ec.png)

[explicit life](https://www.pathofexile.com/trade/search/Expedition/qLGKvvPCg)
![image](https://user-images.githubusercontent.com/3137623/131978243-0b33856c-cfb5-4350-b1b7-d4af28d6a524.png)
